README.md

```diff
@@ -7,20 +7,22 @@ MOLGENIS is a collaborative open source project on a mission to generate great s
 See http://molgenis.github.io for documentation.
 
 ## Installation
-See [Getting Started Guide](http://molgenis.github.io/develop/start).
+See [Getting Started Guide](docs/develop/start.md).
```

Comment:
If I look at how others do this, they actually just link to the MD file on GitHub, and not to the GitBook.

Don't how we want to do this. Easy fix is https://gitbook.com/@molgenis/guide-quickstart.md or something like this